modules/mobile/spec/request/v1/messages_request_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../support/iam_session_helper'
+require_relative '../../support/mobile_sm_client_helper'
+
+RSpec.describe 'Mobile Messages Integration', type: :request do
+  include Mobile::MessagingClientHelper
+  include SchemaMatchers
+
+  let(:user_id) { '10616687' }
+  let(:inbox_id) { 0 }
+  let(:message_id) { 573_059 }
+  let(:va_patient) { true }
+
+  before do
+    allow_any_instance_of(MHVAccountTypeService).to receive(:mhv_account_type).and_return(mhv_account_type)
+    allow(Mobile::V0::Messaging::Client).to receive(:new).and_return(authenticated_client)
+    iam_sign_in(build(:iam_user, iam_mhv_id: '123'))
+  end
+
+  context 'Premium User' do
+    let(:mhv_account_type) { 'Premium' }
+
+    describe '#thread' do
+      let(:thread_id) { 573_059 }
+
+      it 'responds to GET #thread' do
+        VCR.use_cassette('sm_client/messages/gets_a_message_thread') do
+          get "/mobile/v0/messaging/health/messages/#{thread_id}/thread", headers: iam_headers
+        end
+
+        expect(response).to be_successful
+        expect(response.body).to be_a(String)
```

Comment:
I know you're copying this from Patrick's specs, but idk what the value is in confirming it's a string. Confirming that it's not null might be helpful but response body should always be a string on successful requests. 

Also, I don't fully trust schema specs like the one below. I've been burned too many times. Maybe we could replace the string test with something a little more concrete.